<launch>
    <arg name="USE_RVIZ" default="false"/>

 <node pkg="potbot" name="potbot_pp" type="potbot_pp" output="log">
    <param name="USE_RVIZ" value="$(arg USE_RVIZ)"/>
    <param name="PATH_PLANNING_METHOD" value="potential_method"/>
    <!--potential_method csv-->
    <param name="PATH_PLANNING_FILE" value="$(find potbot)/path/-sin.csv"/>
    <param name="USE_AMCL" value="false"/>
    <param name="PID_CONTROL" value="false"/>
    <param name="ANGLE_CORRECTION" value="true"/>
    <param name="MAX_VELOCITY" value="0.5"/>
    <param name="MAX_ANGULAR" value="3.141592653589793"/>
    <param name="TARGET_POSITION_X" value="6.0"/><!--DESTINATION  -9.5, -8.0-->
    <param name="TARGET_POSITION_Y" value="0.0"/>
    <param name="TARGET_POSITION_MARGIN" value="0.3"/>
    <param name="GAIN_PROPORTIONAL" value="1.0"/>
    <param name="GAIN_INTEGRAL" value="10.0"/>
    <param name="GAIN_DIFFERENTIAL" value="0.001"/>
    <param name="AVOIDANCE_RADIUS" value="0.4"/>
    <param name="PATH_CREATE_PERIOD" value="1.5"/>
    <param name="POTENTIAL_FIELD_WIDTH" value="2"/>
    <param name="POTENTIAL_FIELD_DIVDE_X" value="20"/>
    <param name="POTENTIAL_FIELD_DIVDE_Y" value="20"/>
    <param name="CALCULATE_BIAS" value="false"/>
    <param name="POTENTIAL_BIAS_COEFFICIENT_0" value="-1.0"/>
    <param name="POTENTIAL_BIAS_COEFFICIENT_1" value="0.5"/>
    <param name="AHEAD_PATH" value="0.3"/>
    <param name="EXCLUDE_LRF" value="0.05"/>
 </node>
</launch>
